Source/Classes/AnimatedImages/PINCachedAnimatedImage.m

```diff
@@ -351,10 +379,12 @@ - (CFTimeInterval)durationAtIndex:(NSUInteger)index
 
 - (BOOL)playbackReady
 {
-    __block BOOL playbackReady = NO;
-    [_lock lockWithBlock:^{
-        playbackReady = _playbackReady == 0;
-    }];
+    static BOOL playbackReady = NO;
```

Comment:
@garrettmoon I think this might be a bug - the static will be the same variable for all instances, but this needs to be different per instance right?

Reply:
Man… such a dumb mistake. Will put a fix up.